File: externals/Async-CTP/samples/VB/Windows/Phone/Concert/Finder/Concert/Finder/ViewModel/VenueDetailsPageViewModel.vb
Namespace ConcertFinder.ViewModel

    ''' <summary>
    ''' The view model for the VenueDetailsPage.
    ''' </summary>
    Public Class VenueDetailsPageViewModel
        Inherits AbstractDetailsPageViewModel

        Public Sub New()
            Me.EventGroup = Model.EventGroup.Venue
        End Sub

    End Class

End Namespace

File: externals/Async-CTP/samples/CS/Windows/Phone/Concert/Finder/Concert/Finder/ViewModel/VenueDetailsPageViewModel.cs
using ConcertFinder.Model;

namespace ConcertFinder.ViewModel
{
    /// <summary>
    /// The view model for the VenueDetailsPage.
    /// </summary>
    public class VenueDetailsPageViewModel : AbstractDetailsPageViewModel
    {
        public VenueDetailsPageViewModel()
        {
            this.EventGroup = EventGroup.Venue;
        }
    }
}
